File: problemsets/Codeforces/Java/A935.java
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
import java.util.*;

public class A935 {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int n = in.nextInt(), ret = 0;
		for (int i = 1; i <= n/2; i++)
			if ((n-i) % i == 0) ret++;
		System.out.println(ret);
	}
}

File: problemsets/Codeforces/Python/A935.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
n = int(input())
r = 0
for x in range(1,n//2+1):
  r += n%x==0
print(r) 